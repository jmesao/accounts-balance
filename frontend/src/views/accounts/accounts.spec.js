import { createLocalVue, shallowMount } from '@vue/test-utils';

import AccountsView from './accounts.vue';

describe('AccountsView', () => {
  function mountAccountsView() {
    const localVue = createLocalVue();

    return shallowMount(AccountsView, {
      localVue,
    });
  }

  it('should render correctly', () => {
    const wrapper = mountAccountsView();
    expect(wrapper.element).toMatchSnapshot();
  });
});
