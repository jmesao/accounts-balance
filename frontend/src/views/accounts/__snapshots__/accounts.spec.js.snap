// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountsView should render header without items on the accounts balance table 1`] = `
<div>
  <header-container-stub
    title="Accounts"
  >
    <div>
      Home / 
      <span
        class="tw-body-strong"
      >
        Accounts
      </span>
    </div>
  </header-container-stub>
   
  <accounts-balance-table-stub
    items=""
  />
</div>
`;
